server_environment/tests/common.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018 Camptocamp (https://www.camptocamp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import os
+from contextlib import contextmanager
+from unittest.mock import patch
+
+from odoo.tests import common
+from odoo.addons.server_environment import server_env
+from odoo.tools.config import config
+
+import odoo.addons.server_environment.models.server_env_mixin as \
+    server_env_mixin
+
+
+class ServerEnvironmentCase(common.TransactionCase):
```

Comment:
SavepointCase not an option? I see records created for each tests. 
Not a big deal but we some time in our already big builds :innocent:

Reply:
Done!